<p>HTL <code>&lt;sly&gt;</code> elements are automatically unwrapped.</p>
<ul>
    <li>variable names</li>
    <li>template names</li>
</ul>
<h3>Noncompliant Code Example</h3>
<pre>
&lt;sly data-sly-test=&quot;${myTest}&quot; data-sly-unwrap&gt;Caption&lt;/sly&gt;
</pre>
<h3>Compliant Solution</h3>
<pre>
&lt;sly data-sly-test=&quot;${myTest}&quot;&gt;Caption&lt;/sly&gt;

&lt;sly data-sly-test=&quot;${myTest}&quot; data-sly-unwrap=&quot;${false}&quot;&gt;Caption&lt;/sly&gt;
</pre>
