<p>HTL expressions in HTML comments should have defined context.</p>
<h3>Noncompliant Code Example</h3>
<pre>
&lt;!--[if IE]&gt;&lt;link rel=&quot;shortcut icon&quot; href=&quot;${site.root}/images/favicon/favicon.ico?v2&quot;&gt;&lt;![endif]--&gt;
</pre>
<h3>Compliant Solution</h3>
<pre>
&lt;!--[if IE]&gt;&lt;link rel=&quot;shortcut icon&quot; href=&quot;${site.root @ context=&#39;uri&#39;}/images/favicon/favicon.ico?v2&quot;&gt;&lt;![endif]--&gt;
</pre>
