<p>Always use HTL style of commenting</p>
<h3>Noncompliant Code Example</h3>
<pre>
    &lt;!-- Potentially bad, plain HTML comments will be visible on the page --&gt;
</pre>
<h3>Compliant Solution</h3>
<pre>
    &lt;!--/* Good, this comment will not be rendered */--&gt;
</pre>
